chrome-launcher.ts

```diff
@@ -270,7 +275,7 @@ export class Launcher {
               resolve();
             })
             .catch(err => {
-              if (retries > 10) {
+              if (retries > launcher.maxConnectionRetries) {
```

Comment:
Hi and thanks for the PR. I think this is a great addition!

Reply:
Hi Sam, thanks!